src/lib/db/index.ts

```diff
@@ -201,6 +202,7 @@ export const createStores = (
         releasePlanMilestoneStore: new ReleasePlanMilestoneStore(db, config),
         releasePlanMilestoneStrategyStore:
             new ReleasePlanMilestoneStrategyStore(db, config),
+        featureLinkStore: new FakeFeatureLinkStore(),
```

Comment:
A `TODO` or `FIXME` in the code could help indicate what should be replaced in next PRs

Reply:
I will followup instantly today